core/src/main/java/com/linecorp/armeria/server/ServerConfig.java

```diff
@@ -263,6 +264,11 @@ default boolean shutdownBlockingTaskExecutorOnStop() {
      */
     Function<? super ProxiedAddresses, ? extends InetSocketAddress> clientAddressMapper();
 
+    /**
+     * Returns whether {@link WebSocketService} is added to the {@link Server}.
+     */
+    boolean hasWebSocketService();
```

Comment:
I find this method is used only for enabling CONNECT support in an HTTP/2 connection. What happens if we remove this method and always allow CONNECT in HTTP/2, like we did before?

I feel like this method should be removed because it is only useful for ourselves but not very much for our users. Can we:

- just retain the old behavior; or
- let `Server` calculate and keep this property and pass it to `HttpServerHandler`?